src/Console/Command/CompileCommand.php

```diff
@@ -180,9 +180,14 @@ private function buildPHP(string $spcBinaryPath, mixed $phpExtensions, mixed $ar
                 '--build-micro',
                 '--with-micro-fake-cli',
                 '--arch=' . $arch,
+                '--debug',
```

Comment:
This add a lot of output, but i believe this is better in task like this otherwise we are blind when this fail on the CI

Reply:
I would adapt the value according to the output verbosity level

```
if ($output->isVerbose()) {...}
```